### PR TITLE
Add deploy grafana

### PR DESCRIPTION
**Pull request by eranra**

### CODE REVIEW

**Review by Red-GV — COMMENTED** [review]
What is the objective here in adding Grafana to the deployment? 

I'm not seeing where visualizing the log lines are helpful nor possibly replaced by new metric or something.

**Comment by eranra** [comment]
@Red-GV this is hacking code to help understand if the benchmark is working as expected (==debug)

using promQL it is easy to observe the status of the benchmark ... I saw multiple things like "rate of logs" (using the rate logql keyword, I discovered that the rate is wrong and not what I expected)  ... content of logs (making sure we have content diversity in the log lines we create, I was asking my self if there is compression, etc ... and this helped to verify that we don't)  ... also connecting something like Grafana immediately show if Loki world (for me, it showed that I was not connected to S3 bucket initially when deploying)

So ... this is like connecting a "debugger" to the benchmark code and this script is an easy way to achieve that and connect grafana to the Loki backend.

File: hack/ocp-deploy-grafana.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+set -eou pipefail
+
+NAMESPACE="${1:-observatorium-logs-test}"
+TENANT_ID="${2:-observatorium}"
+LOKI_PREFIX="${3:-observatorium-loki}"
+
+# Deploy grafana and expose as route
+kubectl --ignore-not-found=true -n "$NAMESPACE" delete deployment grafana
+kubectl create deployment -n "$NAMESPACE" grafana --image=docker.io/grafana/grafana
+oc --ignore-not-found=true -n "$NAMESPACE" delete service grafana
```
Comment (ronensc) [inline hack/ocp-deploy-grafana.sh:12]: What's the reason for the inconsistent usage of `kubectl` and `oc`?

Comment (eranra) [reply hack/ocp-deploy-grafana.sh]: > What's the reason for the inconsistent usage of `kubectl` and `oc`?

@ronensc no reason ... I moved all to be oc :-)

File: README.md [hunk 2]
```diff
@@ -48,6 +48,8 @@ Upon benchmark execution completion, results are available in the `reports/date+
 
 To uninstall Loki and the Prometheus configuration, run: `make obs-benchmarks-cleanup`. To reconfigure the Loki deployment, change the parameters file and run `make obs-benchmarks-cleanup && make deploy-obs-benchmarks`.
 
+Note: If required, use [hack/ocp-deploy-grafana.sh](hack/ocp-deploy-grafana.sh) to deploy grafana and visualize log lines
```
Comment (eranra) [inline README.md:51]: Move that into a separate section and explain some logQL examples

**Review by Red-GV — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (eranra) [reply README.md]: fixed

**Pull request merged**